contrib/insights/src/column_statistics.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You
+ * may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is
+ * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+ * ANY KIND, either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+#pragma once
+
+#include "mlio.h"
+#include "hll/hyperloglog.h"
+
+namespace insights {
+namespace detail {
+
+static const uint8_t CARDINALITY_HLL_SIZE = 16;
+class MLIO_API column_analysis {
```

Comment:
You can remove `"mlio.h"` and `MLIO_API`. We do not expose this class in our C++ API.